src/models/webhook-notification.ts

```diff
@@ -271,6 +282,27 @@ export class WebhookObjectData extends Model
   constructor(props?: WebhookObjectDataProperties) {
     super();
     this.initAttributes(props);
+    if (
+      this.metadata &&
+      (this.object === 'message' || this.object === 'thread')
```

Comment:
Not a blocker, but I'm wondering if it'll make sense to create a string enum for handling object types  in the future.

Reply:
@ozsivanov that's a good shout, I'll merge it in as is and open a new ticket to make a string enum, thank you 😄